content/post/prioritization_meeting.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Prioritizing Prioritization"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
```

Comment:
Cover image

Reply:
Will update when Margarita sends me the illustration. She's working on it.